Spectacle/Resources/Localizations/it.lproj/Spectacle.strings

```diff
@@ -1,111 +1,111 @@
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Center"; ObjectID = "xJu-Uo-nZw"; */
-"xJu-Uo-nZw.title" = "Center";
+"xJu-Uo-nZw.title" = "Centro";
 
 /* Class = "NSMenuItem"; title = "Fullscreen"; ObjectID = "dwQ-Bl-nEL"; */
-"dwQ-Bl-nEL.title" = "Fullscreen";
+"dwQ-Bl-nEL.title" = "Intero schermo";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Left Half"; ObjectID = "Jae-vx-7ZC"; */
-"Jae-vx-7ZC.title" = "Left Half";
+"Jae-vx-7ZC.title" = "Metà sinistra";
 
 /* Class = "NSMenuItem"; title = "Right Half"; ObjectID = "cae-26-Oab"; */
-"cae-26-Oab.title" = "Right Half";
+"cae-26-Oab.title" = "Metà destra";
 
 /* Class = "NSMenuItem"; title = "Top Half"; ObjectID = "yh0-ik-weD"; */
-"yh0-ik-weD.title" = "Top Half";
+"yh0-ik-weD.title" = "Metà alta";
 
 /* Class = "NSMenuItem"; title = "Bottom Half"; ObjectID = "huV-Gw-lbX"; */
-"huV-Gw-lbX.title" = "Bottom Half";
+"huV-Gw-lbX.title" = "Metà bassa";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Upper Left"; ObjectID = "ldc-QO-Cq6"; */
-"ldc-QO-Cq6.title" = "Upper Left";
+"ldc-QO-Cq6.title" = "In alto a sinistra";
 
 /* Class = "NSMenuItem"; title = "Lower Left"; ObjectID = "eN5-A6-iho"; */
-"eN5-A6-iho.title" = "Lower Left";
+"eN5-A6-iho.title" = "In basso a sinistra";
 
 /* Class = "NSMenuItem"; title = "Upper Right"; ObjectID = "C7t-Sf-kOI"; */
-"C7t-Sf-kOI.title" = "Upper Right";
+"C7t-Sf-kOI.title" = "In alto a destra";
 
 /* Class = "NSMenuItem"; title = "Lower Right"; ObjectID = "oDU-J0-NkJ"; */
-"oDU-J0-NkJ.title" = "Lower Right";
+"oDU-J0-NkJ.title" = "In basso a destra";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Next Display"; ObjectID = "I4U-jL-hXq"; */
-"I4U-jL-hXq.title" = "Next Display";
+"I4U-jL-hXq.title" = "Schermata Successiva";
 
 /* Class = "NSMenuItem"; title = "Previous Display"; ObjectID = "aPo-Rp-A1X"; */
-"aPo-Rp-A1X.title" = "Previous Display";
+"aPo-Rp-A1X.title" = "Schermata Precedente";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Next Third"; ObjectID = "7HO-L3-O6Q"; */
-"7HO-L3-O6Q.title" = "Next Third";
+"7HO-L3-O6Q.title" = "Terza Successiva";
 
 /* Class = "NSMenuItem"; title = "Previous Third"; ObjectID = "bV9-Ga-jDG"; */
-"bV9-Ga-jDG.title" = "Previous Third";
+"bV9-Ga-jDG.title" = "Terza Precedente";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Make Larger"; ObjectID = "Yun-AA-OVD"; */
-"Yun-AA-OVD.title" = "Make Larger";
+"Yun-AA-OVD.title" = "Ingrandisci";
 
 /* Class = "NSMenuItem"; title = "Make Smaller"; ObjectID = "gTe-ym-DB0"; */
-"gTe-ym-DB0.title" = "Make Smaller";
+"gTe-ym-DB0.title" = "Riduci";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Undo"; ObjectID = "l9O-0B-fjp"; */
-"l9O-0B-fjp.title" = "Undo";
+"l9O-0B-fjp.title" = "Annulla";
 
 /* Class = "NSMenuItem"; title = "Redo"; ObjectID = "187-7k-0Ct"; */
-"187-7k-0Ct.title" = "Redo";
+"187-7k-0Ct.title" = "Ripristina";
 
 /* --- */
 
 /* Class = "NSMenuItem"; title = "Restore Defaults..."; ObjectID = "aFM-wq-kOy"; */
-"aFM-wq-kOy.title" = "Restore Defaults...";
```

Comment:
You can probably delete this too. Odds are I missed removing this when I originally created these files.